### ISSUE TITLE
The response from users followings has a different structure than the rest thumb images of the other requests

### ISSUE THREAD

**Issue by PSilva2351** [audioBoom/api#4]

**Comment by harrane** [comment]
From user followings, the thumb for a channel is:
"logo_image": {
          "original": "https://d15mj6e6qmt1na.cloudfront.net/i/15626152",
          "thumb": "https://d15mj6e6qmt1na.cloudfront.net/i/15626152/150x150"
        },

From request a channel specifically, the thumb is:
"logo_image": {
          "original": "https://d15mj6e6qmt1na.cloudfront.net/i/14756064",
          "thumb": {
            "width": 150,
            "height": 150,
            "url": "https://d15mj6e6qmt1na.cloudfront.net/i/14756064/150x150"
          }
        },

Could this be a difference of the api version?

**Comment by jdelStrother** [comment]
Yeah, we were finding that we never used the width & height, so they were omitted from the v2 api.  They're not so relevant with the dynamic image-resizing stuff.
